public/main/client/crypto/aes256cbc.js

```diff
@@ -3,6 +3,7 @@
 const crypto = require('crypto')
 
 function decrypt (password, data) {
+  // eslint-disable-next-line node/no-deprecated-api
```

Comment:
see Note in Readme for this

Reply:
If it still works, let's address the deprecation in another issue/PR.

Reply:
I created a ticket #486